src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,195 @@
+  - Welcome to New Relic
+  - Get started
+translate:
+  - jp
+  - kr
+metaDescription: 'What is New Relic? No matter what is your business or role, learn how to make the most of our suite of digital intelligence products.'
+redirects:
+  - /docs/accounts-partnerships/education/getting-started-new-relic/introduction-new-relic
+  - /docs/accounts-partnerships/getting-started-new-relic/introduction-new-relic
+  - /docs/using-new-relic/welcome-new-relic/getting-started/introduction-new-relic
+  - /taxonomy/term/5766
+  - /docs/using-new-relic/welcome-new-relic/getting-started/what-new-relic-can-do-you
+  - /docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-one-transition-guide-july-2020
+  - /docs/using-new-relic/welcome-new-relic/get-started/aW50cm9kdW
+  - /docs/using-new-relic/welcome-new-relic/get-started/introduction-new-relic
+  - /docs/using-new-relic/
+  - /docs/using-new-relic/welcome-new-relic
+  - /docs/using-new-relic/welcome-new-relic/get-started/intro-new-relic
+  - /docs.newrelic.com/docs/new-relic-solutions/new-relic-one/ui-data/new-relic-feature-end-life-announcements-july-2020
+  - /docs/new-relic-solutions/new-relic-one/install-configure/install-new-relic
+  - /docs/new-relic-solutions/get-started/quick-launch-guide
+  - /docs/using-new-relic/welcome-new-relic/get-started/get-started-full-stack-observability
+  - /docs/agents/manage-apm-agents/installation/install-agent
+  - /docs/new-relic-one/use-new-relic-one/cross-product-functions/install-configure/install-new-relic
+  - /docs
+  - /docs/using-new-relic/cross-product-functions/install-configure/install-new-relic
+signupBanner:
+  text: Monitor and improve your entire stack. 100GB free. Forever.
+freshnessValidatedDate: never
+---
+
+import NROneMain from 'images/new-relic-one_screenshot-full_main.webp'
+
+import NRAlertsMain from 'images/alerts_screenshot-crop_main.webp'
+
+import NRApmMain from 'images/apm_screenshot-crop_main.webp'
+
+import NRBrowserMain from 'images/browser_screenshot-crop_main.webp'
+
+
+
+
+Eureka! That's the feeling you'll have when you learn how New Relic works. 
+With New Relic, you can monitor the performance of your applications, infrastructure, web browsers, and more. You can do this on a single platform with real-time data and analytics. You will identify any issues that may be impacting its functionality. It enables you to make informed decisions and optimize your systems for better performance.
+
+Let us help you and your team make your life better:
+
+  * Get full-stack observability across your enterprise to predict and detect issues before they impact customers.
+
+  * Put all your telemetry data, including open-source, on one secure scalable database with 700+ integrations.
+
+  * Visualize and analyze your entire stack in one unified experience, with AI-powered analytics to find and fix issues fast.
+
+  * Pay only for what you use.
+
+<img
+  title="New Relic main page"
+  alt="New Relic main page"
+  src={NROneMain}
+/>
```

Comment:
I'd pull this image up higher. Also, I'm wondering if there's a more exciting/interesting/intriguing image we could use?